src/Log/EchoLogger.php

```diff
@@ -17,71 +17,71 @@ class EchoLogger implements LoggerInterface
     /**
      * @inheritDoc
      */
-    public function emergency($message, array $context = array())
+    public function emergency(string|\Stringable $message, array $context = []): void
```

Comment:
Bitte das Stringable Interface in den uses aufnehmen.

Reply:
Die Signatur kommt so aus dem PSR Interface und würde das hier ungern verändern